### PR TITLE
STORM-3159: Fixed a potential file resource leak.

### PR DESCRIPTION
**Pull request by zd-project**
The file is not properly put in the try-with-resource/try-finally block.

### CODE REVIEW

**Comment by zd-project** [comment]
This will affect the implementation of #2754

**Comment by danny0405** [comment]
So, in what case the file resource leak will leak ?

**Comment by zd-project** [comment]
The file resource was not wrapped in try-finally or try-with-resource block. In case of IOException, resource may not be closed properly

File: storm-server/src/main/java/org/apache/storm/utils/ServerUtils.java [hunk 1]
```diff
@@ -635,16 +635,16 @@ public static void unZip(File inFile, File toDir) throws IOException {
      *
      * @throws IOException
      */
+    //Used only by logviewer, considering moving to web-app?
```
Comment (agresch) [inline storm-server/src/main/java/org/apache/storm/utils/ServerUtils.java]: even if used only in one place, if it is a general purpose function, I think it makes sense to have in a global place.  I'd remove this comment.

**Comment by srdo** [comment]
+1. Please raise an issue for this, and squash to one commit that contains the issue number.

**Pull request merged**